variables.tf

```diff
@@ -44,6 +44,16 @@ variable "bucket_name" {
   description = "Name of the S3 bucket to create"
 }
 
+variable "default_root_object" {
+  type = string
+
+  # Distributions without default_root_object return an empty string,
+  # so it's best to mimic that behavior to avoid diffs later on.
+  default = ""
```

Comment:
I am not sure if this is required. I think it should also work with `null`

Reply:
Yes, let's use `null` as default to ensure we don't introduce a breaking change.